service_container/tags.rst

```diff
@@ -570,47 +570,113 @@ application handlers::
         }
     }
 
-.. tip::
+Prioritizing Tagged Services
```

Comment:
Maybe „Tagged Services with Priority“ sounds more natural?

cc @javiereguiluz

Reply:
For me the current one sounds a bit clearer, but if others think that "Tagged Services with Priority" is better I will just update it.

Reply:
Changed it to how you suggested